source/features/dim-bots.tsx

```diff
@@ -4,15 +4,24 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-const commitSelector = [
-	/* Commits */
-	'.commit-author[href$="%5Bbot%5D"]',
-	'.commit-author[href$="renovate-bot"]',
-	'.commit-author[href$="scala-steward"]',
-].join(',');
+const commitSelectors = [
+	'actions-user',
+	'bors',
+	'imgbot',
+	'octomerger',
+	'renovate-bot',
+	'rust-highfive',
+	'rust-lang',
+	'scala-steward',
+	'snyk-bot',
+	'web-flow',
+].map(bot => `.commit-author[href$="?author=${bot}"]`);
```

Comment:
Including the `?author=` part ensures we match the exact bot name (prefixing with a slash doesn't work in this case).